tests/test_freshdesk_bookmarks.py

```diff
@@ -0,0 +1,218 @@
+"""Test tap check mode and metadata/annotated-schema."""
```

Comment:
This docstring is copy pasta, can be deleted

Reply:
Deleted